### PR TITLE
App breaks when opening help

### PR DESCRIPTION
**Pull request by minna-xD**
I happened to notice the help didn't open anymore (for any language) after your latest change to remove SupportedCultures. HelpForm still references the attribute.

Here's my super simple fix if it's acceptable. (I'm a complete c# newbie, my husband helped me fix it based on my solution idea  :)  )

### CODE REVIEW

**Comment by alxnull** [comment]
@minna-xD Great catch, I didn't notice that! 👍 

There might be another case to consider for the solution: If `CurrentUICulture` is set to a language not supported by the app (which can happen), we need a fallback to English (similar to the rest of the UI). The simplest options could be to either check if a file `Help.<lang-code>.html` exists in the `Help` folder _or_ if the current Culture is part of `AppInfo.GetSupportedCultures()` (which doesn't depend on the removed attribute). If that's not the case, we would load the English help file.

Would be great if you could try to implement this as well (in case you want to play around with C# a little more 😃). I can help if you need any further guidance.

**Comment by minna-xD** [comment]
Sure, I'll give it a go! I was originally thinking about checking if a help file exists and then falling back to English if not, so I'll try that.

**Comment by minna-xD** [comment]
I made a new commit yesterday (letting you know in case there's no automatic notification)

File: ScreenRuler/Forms/HelpForm.cs [hunk 1]
```diff
@@ -15,8 +15,15 @@ public HelpForm()
...
             InitializeComponent();
             this.KeyPreview = true;
-            // Set lang code
+            // Set lang and check if there is a help file for it.
+            // If not, notify user and fall back to English.
             lang = CultureInfo.CurrentUICulture.TwoLetterISOLanguageName;
+            if (!File.Exists(HelpFileLocation))
+            {
+                var languageName = CultureInfo.CurrentUICulture.DisplayName;
+                MessageBox.Show($"No help file found for {languageName}. Showing English help.", "Language-specific help file not found");
```
Comment (alxnull) [inline ScreenRuler/Forms/HelpForm.cs]: I think we don't need this extra message box here. If this fallback happens, the rest of the UI usually will be in English already anyway.

**Review by alxnull — COMMENTED** [review]
Awesome! Looks good to me, just one comment below.

File: ScreenRuler/Forms/HelpForm.cs (hunk 1 above)
Comment (minna-xD) [reply ScreenRuler/Forms/HelpForm.cs]: OK! This was my husband's suggestion and I was also thinking of a situation where a help file hasn't yet been translated (even if the UI was), but then again, the help file is so small that it's effortless to translate along with everything else.
I just did a new commit removing the messagebox.

**Review by alxnull — APPROVED** [review]
Perfect, thanks again for your help! 😃

**Pull request merged**